swagger.yml

```
@@ -0,0 +1,137 @@
+swagger: '2.0'
+
+# This is your document metadata
+info:
+  version: "0.0.0"
+  title: Communication_Notebook
+
```

このドキュメント自体のdescription書きましょう

```suggestion
  description: ○○のAPI仕様です。

```